docs/user-guide/configuration-files/strongbox-authorization-yaml.md

```diff
@@ -0,0 +1,51 @@
+# General
```

Comment:
@fuss86 ,

Please, don't use "General" here (on the first line) as this will be the page's title and the indexer will index all these pages as being called "General". You can have a look at the notes on how to write pages for the new wiki [here](https://github.com/strongbox/strongbox-docs).

Thanks! :)

Reply:
@carlspring Good catch! Yeah, this should be `# Strongbox Authorization Yaml` or something like that.

Reply:
Updates committed :)